src/components/card/index.tsx

```diff
@@ -70,6 +70,7 @@ export const Card = (props: ICardProps) => {
     marginBottom: '16px'
   };
   const defaultButtonStyles: CSSProperties = {
+    maxWidth: 'calc(50% - 32px)',
```

Comment:
same here

Reply:
We've added a margin for the button, so using a percentage might not function correctly in this context.